_scripts/instruction-widget/install.js

```diff
@@ -98,6 +98,8 @@ module.exports = function(context) {
   debian_install = function() {
     template = "debian";
     context.devuan = context.distro == "devuan"
+    context.jessie = context.version == 8
+    context.stretch = context.version == 9
```

Comment:
It looks like `context.stretch` is being set to true for testing/unstable too causing it to tell you to install Stretch backports on that system too. This is due to: https://github.com/certbot/website/blob/3c8502138130f49a10bd95b6d33aaa2a51782cf3/_scripts/instruction-widget/data/inputs.json#L53 We should update this to a new value to avoid this problem now and in the future.